### ISSUE TITLE
Update `FromIRToExecutable.md` (`csv-to-ld-options` no longer exists)

### ISSUE THREAD

**Issue by paleozogt** [revng/revng#50]
I'm going through [FromIRToExecutable](https://github.com/revng/revng/blob/master/docs/FromIRToExecutable.rst), where it mentions using `csv-to-ld-options`:

```
gcc $(csv-to-ld-options translated.ll.li.csv) \
    translated.o \
    -lz -lm -lrt \
    -o translated.elf
```

However, I don't see this in the orchestra distribution.  Has the tool been renamed?

**Comment by aleclearmind** [comment]
This is now embedded in `revng translate`. You can run `revng --verbose translate ...` to see what gets invokes.
What's the use case for using it on its own?

**Comment by paleozogt** [comment]
The use case is that I'd like to translate to something other than x86_64.

Thanks for the info! :)

**Comment by aleclearmind** [comment]
It should not be super hard, if you know where to look.
Keep in mind that you need to configure `qemu` to build helpers for your target of choice.

**Comment by paleozogt** [comment]
Are there instructions for how to build revng fully from source (so that I can have it build those helpers)?  The orchestra instructions ([here](https://rev.ng/getting-started.html)) seem out of date.

**Comment by aleclearmind** [comment]
Just add `qemu` to projects to build from source:
https://github.com/revng/orchestra#building-from-source

**Comment by aleclearmind** [comment]
@paleozogt Feel free to join [our discord](https://discord.com/invite/fJF2VrzdCJ), we'd happy to help in porting to non-x86-64 architectures.

**Comment by paleozogt** [comment]
@aleclearmind Thanks for the discord invite.

**Comment by aleclearmind** [comment]
Fixed in 86d931dc1fd0bfe329eed414936167bf1243c287.
Thanks!
